requirements/base.txt

```diff
@@ -162,7 +162,7 @@ multidict==5.1.0
     # via
     #   aiohttp
     #   yarl
-numpy==1.21.1
+numpy==1.22.2
```

Comment:
This needed to be bumped to resolve https://github.com/numpy/numpy/issues/19033. Note this was bumped by deleting lines 165–168 and running `pip-compile-multi --no-upgrade`.